presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -92,6 +92,9 @@
     @Option(name = "--password", title = "password", description = "Prompt for password")
     public boolean password;
 
+    @Option(name = "--use-okta", title = "Okta login", description = "OpenID login with Okta")
+    public boolean useOkta;
+
```

Comment:
I am thinking rather than being option of getting this passed in, we should rather enforce this option.

Reply:
For our version looks like a good idea. Also URL should be in the code instead of passing

Reply:
Made this `true` as default.